projects/kit/components/like/like.style.less

```diff
@@ -0,0 +1,88 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+@keyframes tuiLikeScale {
+    0% {
+        transform: scale(1);
+    }
+
+    50% {
+        transform: scale(1.4);
+    }
+
+    100% {
+        transform: scale(1.1);
+    }
+}
+
+/**
+ * @name Like
+ * @selector [tuiLike]
+ *
+ * @description
+ * A stylized input type="checkbox"
+ *
+ * @attributes
+ * data-size — size (default: 'm') ('s' | 'm')
+ *
+ * @vars
+ * --t-icon-color - custom color for `:checked` state
+ * --t-unchecked-icon — default state icon
+ * --t-checked-icon — checkmark icon
+ *
+ * @example
+ * <input tuiLike />
+ * <input tuiLike="var(--tui-status-negative)" />
```

Comment:
Let's not use this example because this is done with Angular, not with pure CSS

Reply:
removed